python/lsst/source/injection/utils/_make_injection_pipeline.py

```diff
@@ -329,4 +340,35 @@ def make_injection_pipeline(
         len(injected_subset_labels),
         grammar2,
     )
+
+    # Optionally incluce additional tasks in the injection pipeline.
+    if input_pipelines:
```

Comment:
maybe, to be consistent, in this section:
`input_pipelines` -> `additional_pipelines`
and
`input_pipeline` -> `additional_pipeline`